### PR TITLE
feat(accounts): add accounts core concept

### PR DESCRIPTION
**Pull request by Jacob-CG**

### CODE REVIEW

File: docs/core-concepts/accounts.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+### Rent
+
+Storing data on accounts costs money to maintain, and it is funded by what is called 
+rent. If you maintain a minimum balance equivalent to 2 years of rent payments in an 
+account, your account will be exempt from paying rent.
+
+Rent is paid during two different timings:
+
+1. When references by a transaction
```
Comment (arcticmatt) [inline docs/core-concepts/accounts.md:67]: should this be "referenced"

File: docs/core-concepts/accounts.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+must create two accounts, one account to store the program's code, and one to store 
+the counter.
+
+![](./account_example.jpeg)
+
+To prevent an account from being deleted, you must pay rent.
+
+### Rent
+
+Storing data on accounts costs money to maintain, and it is funded by what is called
```
Comment (arcticmatt) [inline docs/core-concepts/accounts.md:61]: could be interesting to mention that you can withdraw rent. e.g. this is common for candy machines after the mint has finished

File: docs/core-concepts/accounts.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+## Deep Dive
+
+### Account Model
+
+There are 3 kinds of accounts on Solana:
+
+- Data accounts store data
+- Program accounts store executable programs
+- Native accounts that indicate native programs on Solana such as System, Stake, and Vote
```
Comment (arcticmatt) [inline docs/core-concepts/accounts.md:29]: Could add something like this:

> If you're coming from Ethereum, it's important to note that Solana separates data and code! While Ethereum contracts are stored alongside the data they maintain, Solana programs read/write data to and from independent data accounts.

Comment (Jacob-CG) [reply docs/core-concepts/accounts.md:29]: We are generally trying to not make references to Ethereum within the cookbook. We may make a guide separate to address the differences, but leave out any Ethereum specific language in the meat of the cookbook.

File: docs/core-concepts/accounts.md (hunk 1 above)
Comment (Jacob-CG) [reply docs/core-concepts/accounts.md]: fixed in 66f1d87914d01c9d854c551a0f81cd00f1919d40

File: docs/core-concepts/accounts.md (hunk 2 above)
Comment (Jacob-CG) [reply docs/core-concepts/accounts.md:68]: Added in 66f1d87914d01c9d854c551a0f81cd00f1919d40

**Pull request merged**